L/LLVMLibUnwind/LLVMLibUnwind@18.1.7/build_tarballs.jl

```diff
@@ -0,0 +1,7 @@
+version = v"18.1.7"
+
+include("../common.jl")
+
+# Build the tarballs
+build_tarballs(ARGS, configure(version; experimental=true)...;
+               preferred_gcc_version=v"6.1.0", julia_compat="1.12")
```

Comment:
This Linux AArch64 failure might be because we need GCC 9, IIUC:
```
[18:25:46] /tmp/ccanmniA.s:4876: Error: unknown architecture `armv8.5-a'
[18:25:46]
[18:25:46] /tmp/ccanmniA.s:4877: Error: unknown architectural extension `memtag'
[18:25:46] /tmp/ccanmniA.s:4877: Error: unknown mnemonic `stg' -- `stg x0,[x0]'
```
Based on some brief googling, it appears GCC 9 introduced ARMv8.5 support and `memtag` (dunno about `stg`) requires ARMv8.5.

Reply:
```suggestion
               preferred_gcc_version=v"10", preferred_llvm_version=v"17", julia_compat="1.12")
```
These are the versions we're specifying for LLVM_full v18.1.7, perhaps relevant here as well?

Reply:
I don't know I'm afraid. Try it out?

Reply:
Sure, applied my suggestion. Worst case scenario we can revert the commit.